src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -1436,7 +1443,7 @@ public void testCreateDeleteDatasetLink() {
         createDataverseResponse = UtilIT.createRandomDataverse(apiToken);
         createDataverseResponse.prettyPrint();
         dataverseAlias = UtilIT.getAliasFromResponse(createDataverseResponse);
-        
+        publishTargetDataverse = UtilIT.publishDataverseViaNativeApi(dataverseAlias,  apiToken);        
```

Comment:
It would be nice to assert that the status code is "OK" or whatever it is.

Reply:
sure.